C++/Logger/conversion/Stringifiable.h

```diff
@@ -2,17 +2,20 @@
 #define _STRINGIFIABLE_H_
 
 #include "ConvertibleTo.h"
+#include "../metaprogramming/value_type_is.h"
 
 #include <string>
 
 namespace Toolbox::Log::Conversion
 {
 
-class Stringifiable : public ConvertibleTo<std::string>
+class Stringifiable
+  : public ConvertibleTo<std::string>
+  , value_type_is <std::string>
 {
 public:
   virtual const std::string toString (void) const = 0;
-  operator std::string (void) const override final;
```

Comment:
I think you maybe mistaking overriding with overloading.  Overriding is when a subclass reimplements a method defined in a base class thus providing new functionality.  The override specifier is optional, it adds a compile-time check that the function actually overrides a base method (so if you typed it wrong or the method signature changed it wouldn't compile).  This is helpful for identifying places in the code that need to be updated when you make an interface change and it is also helpful because it allows you to see which methods on a class override their base implementation.